blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,131 @@
+			: inheritGlobalContent;
+	const { content_elements: elements = [] } = showGlobalContent ? globalContent : content;
+
+	if (!elements.length) return null;
+
+	return (
+		<Stack className={BLOCK_CLASS_NAME}>
+			{headerText && headerText.trim() !== "" && (
+				<Heading className={`${BLOCK_CLASS_NAME}__main-title`}>{headerText}</Heading>
+			)}
+
+			{elements && elements.length && (
+				<Carousel
+					id={id}
+					label={phrases.t("story-carousel.aria-label")}
+					nextButton={
+						<Carousel.Button id={id} label={phrases.t("story-carousel.right-arrow-label")}>
+							<Icon name="ChevronRight" />
+						</Carousel.Button>
+					}
+					previousButton={
+						<Carousel.Button id={id} label={phrases.t("story-carousel.left-arrow-label")}>
+							<Icon name="ChevronLeft" />
+						</Carousel.Button>
+					}
+				>
+					{elements.map((item, index) => (
+						<Carousel.Item
+							key={item._id}
+							label={`${phrases.t("story-carousel.slide-indicator", {
+								current: index + 1,
+								maximum: elements.length,
+							})}`}
+						>
+							<a className={`${BLOCK_CLASS_NAME}__story-card`} href={item.canonical_url}>
+								<Image alt={item.headlines.basic} src={extractImageFromStory(item)} />
+								{item.headlines?.basic ? (
+									<Heading
+										className={`${BLOCK_CLASS_NAME}__story-card-header`}
+										truncationLines={itemHeaderTruncationLines}
+										data-testid={itemHeaderTruncationLines}
+									>
+										{item.headlines.basic}
+									</Heading>
+								) : null}
+								{item.description?.basic ? (
+									<Paragraph
+										truncationLines={itemDescriptionTruncationLines}
+										data-testid={itemDescriptionTruncationLines}
+									>
+										{item.description.basic}
+									</Paragraph>
+								) : null}
+							</a>
+						</Carousel.Item>
+					))}
+				</Carousel>
+			)}
+		</Stack>
+	);
+};
+
+StoryCarousel.propTypes = {
+	customFields: PropTypes.shape({
+		carouselContentConfig: PropTypes.contentConfig("ans-feed").tag({
+			group: "Configure Content",
+			label: "Display Content Info",
+		}),
+		inheritGlobalContent: PropTypes.bool.tag({
+			group: "Configure Content",
+			defaultValue: false,
+		}),
+		headerText: PropTypes.string.tag({
+			label: "Headline",
+			default: "",
+			description: "Add a headline for the story carousel component.",
+		}),
+		itemHeaderTruncationLines: PropTypes.number.tag({
+			label: "Carousel item header truncation",
+			default: 0,
+			description:
+				"Number of text lines for the carousel item header to show before being truncated",
+		}),
+		itemDescriptionTruncationLines: PropTypes.number.tag({
+			label: "Carousel item description truncation",
+			default: 0,
+			description:
+				"Number of lines the description in a carousel item will show before being truncated",
+		}),
+	}),
+};
+
+StoryCarousel.label = "Story Carousel - Arc Block";
+
+StoryCarousel.icon = "ui-browser-slider";
```

Comment:
was this slider the pick from product for the icon? not sure if this was in the ticket

Reply:
cool